app/models/product.rb

```diff
@@ -15,6 +15,9 @@ class Product < ActiveRecord::Base
   has_many   :price_policies
   has_many   :training_requests, dependent: :destroy
 
+  cattr_accessor(:types) { ConfigurableArray.new([Instrument, Item, Service, Bundle]) }
```

Comment:
So, if we think we are going to end up refactoring the existing inserts to use ConfigurableArray, or if we foresee reusing it in more places in the future, I'm okay with keeping it. It's certainly more readable than `insert(index, ...)`.

I mostly just felt like adding an entire class for just one or two use cases was a bit overkill when Ruby already has an array class, but if we think we'll end up reusing this I'm fine with it.